src/components/SideBar/index.jsx

```diff
@@ -1,32 +1,30 @@
-import { BsPlus, BsFillLightningFill, BsGearFill } from 'react-icons/bs';
-import { FaFire, FaPoo } from 'react-icons/fa';
+import { BsPlus, BsFillLightningFill, BsGearFill } from "react-icons/bs";
+import { FaFire, FaPoo } from "react-icons/fa";
 
 const SideBar = () => {
   return (
-    <div className="fixed top-0 left-0 h-screen w-16 flex flex-col
-                  bg-white dark:bg-gray-900 shadow-lg">
-                    
-        <SideBarIcon icon={<FaFire size="28" />} />
-        <Divider />
-        <SideBarIcon icon={<BsPlus size="32" />} />
-        <SideBarIcon icon={<BsFillLightningFill size="20" />} />
-        <SideBarIcon icon={<FaPoo size="20" />} />
-        <Divider />
-        <SideBarIcon icon={<BsGearFill size="22" />} />
+    <div
+      className="fixed top-0 left-0 h-screen w-16 flex flex-col
+                  bg-white dark:bg-gray-900 shadow-lg"
+    >
+      <SideBarIcon icon={<FaFire size="28" />} />
+      <Divider />
+      <SideBarIcon icon={<BsPlus size="32" />} />
+      <SideBarIcon icon={<BsFillLightningFill size="20" />} />
+      <SideBarIcon icon={<FaPoo size="20" />} />
+      <Divider />
+      <SideBarIcon icon={<BsGearFill size="22" />} />
     </div>
   );
 };
 
-const SideBarIcon = ({ icon, text = 'tooltip 💡' }) => (
+const SideBarIcon = ({ icon, text = "tooltip 💡" }) => (
   <div className="sidebar-icon group">
     {icon}
-    <span class="sidebar-tooltip group-hover:scale-100">
-      {text}
-    </span>
+    <span className="sidebar-tooltip group-hover:scale-100">{text}</span>
```

Comment:
This line is the major change, the rest changes are versoning and of single quotes changed to double quotes